esp/src/eclwatch/MemberOfWidget.js

```diff
@@ -62,9 +62,7 @@ define([
             this.grid.on("dgrid-datachange", function(event){
                 if (dojoConfig.isAdmin && params.username === context.currentUser && event.oldValue === true) {
                     var msg = confirm(context.i18n.RemoveUser + " " + event.rowId + ". " + context.i18n.ConfirmRemoval);
-                    if (!msg){
-                        event.preventDefault();
-                    }
+                    msg ? location = location.origin : event.preventDefault();
```

Comment:
Putting side-effects inside a "?" operator would be unusual, IOW:
```
if (msg) {
  location = location.origin;
} else { 
  event.preventDefault();
}
```
Would be preferable.

Question:  What does ```location = location.origin``` do?